airbyte-webapp/src/locales/en.json

```diff
@@ -281,6 +281,13 @@
   "sources.emptySchema": "Schema is empty",
   "sources.noSources": "Source list is empty",
   "sources.sourceSettings": "Source Settings",
+  "sources.documentationPanel.tabs.setupGuide": "Setup Guide",
+  "sources.documentationPanel.tabs.schema": "Schema",
+  "sources.documentationPanel.tabs.erd": "ERD (Entities Relationship Diagram)",
```

Comment:
I think ERD is usually "Entity Relationship Diagram" (singular not plural for Entity). Wikipedia writes is as "Entity-relationship diagram", but that feels pretty weird having a hyphen between 1 and 2 but not 3, and since I found a lot of places that write it as "Entity Relationship Diagram" out there, I think we can also skip that hyphen (but still should go with singular, not Entities).